tools/requirements.txt

```diff
@@ -1 +1 @@
-cryptography==36.0.0
+cryptography
```

Comment:
Actually, we have deliberately kept this fixated so as to not break behaviour of any internal functionality. 
As the cryptography updates are not backward compatible and may contain breaking changes.

But yes, This version is too old now and should be updated to the latest one.
I will check if the latest cryptography version breaks anything here.